tools/travis-ci/build.sh

```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
 
-set -euox pipefail
+set -euo pipefail
 
-BUILD_TYPE=${1}
-CMAKE_FLAGS=${2}
+BUILD_TYPE="${1}"
+CMAKE_FLAGS="${2}"
 
-mkdir -p build
-pushd build
+mkdir build && cd build
```

Comment:
mkdir bitte immer mit `-p`

Reply:
und da bereits mit `set -e` ist auch das && nicht notwendig, sogar falsch

Reply:
wenn man nur 1 Ordner erstellt? Das `&&` in dem Fall weil das für mich 1 Befehl ist (bzw 1 Einheit). Falsch ist das nicht. Warum meinst du das?

Reply:
Ah, korrekt ja. Würde das hier aber als feature sehen: Es wird vorausgesetzt der Ordner existiert nicht. Ansonsten müssten wir den vorher entfernen.

Reply:
naja wenn du das lokal ausführst, willst du das nicht jedesmal machen. daher ist es eher ein mkdir -p

Reply:
Hm... von mir aus. Man kann sich ja auf travis verlassen, dass das leer ist